### PR TITLE
Re-enable Tables converter tests

### PR DESCRIPTION
**Pull request by pakrym**
~~Files that were removed are specific to converter implementation. Now that we use the type binder they don't apply.~~

### CODE REVIEW

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/EntityPropertyToTConverterFactoryTests.cs [hunk 1]
```diff
@@ -0,0 +1,907 @@
...
+            IConverter<EntityProperty, bool> converter = EntityPropertyToTConverterFactory.Create<bool>();
+            // Assert
+            Assert.NotNull(converter);
+            EntityProperty property = new EntityProperty(expected);
+            bool actual = converter.Convert(property);
+            Assert.AreEqual(expected, actual);
+        }
+
+        [Test]
+        public void Create_Boolean_ConvertThrowsIfNullProperty()
```
Comment (christothes) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/EntityPropertyToTConverterFactoryTests.cs:44]: nit: these could all be turned into 4 or 5 tests with different types fed to the TestCaseSource.

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TableEntityValueBinderTests.cs [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+            string input = null;
+            // Act
+            EntityProperty property = TableEntityValueBinder.DeepClone(new EntityProperty(input));
+            // Assert
+            Assert.NotNull(property);
+            Assert.AreEqual(EdmType.String, property.PropertyType);
+            Assert.Null(property.StringValue);
+        }
+
+        private class SimpleTableEntity
```
Comment (christothes) [inline sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TableEntityValueBinderTests.cs:331]: Is this used?

**Review by christothes — APPROVED** [review]

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/EntityPropertyToTConverterFactoryTests.cs (hunk 1 above)
Comment (pakrym) [reply sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/EntityPropertyToTConverterFactoryTests.cs:44]: I didn't change this file. I initially removed it and then added it back.

File: sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TableEntityValueBinderTests.cs (hunk 2 above)
Comment (pakrym) [reply sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TableEntityValueBinderTests.cs:331]: I didn't change this file. I initially removed it and then added it back. It's still excluded from the project.

**Pull request merged**